src/Routing/Route/DashedRoute.php

```diff
@@ -56,11 +56,12 @@ protected function _camelizePlugin($plugin)
      * camelBacked form.
      *
      * @param string $url The URL to parse
+     * @param string $method The HTTP method.
      * @return array|false An array of request parameters, or false on failure.
      */
-    public function parse($url)
+    public function parse($url, $method = '')
```

Comment:
@markstory I understand so too. But in the same table, I can see inconsistency:
> Add an argument to an overridden method　No

If only the first overriding is acceptable, then I can understand the intention of the guidelines, and I can say "This is a backword compatible change." though.

This is not only an issue about E_STRICT level errors. If user has an overridden method like the following,
```
public function parse($url, $forTesting = false) {
    if ($forTesting) { // Passing "GET" or "POST" or something, this check will be true.
        // For unit test
    }
    return parent::parse($url);
}
```
the application would be broken by this commit without any E_STRICT level errors. Despite following the guidelines.

@ionas Note that using `func_get_args()` doesn't solve this issue neither.